README.md

```
@@ -408,7 +408,8 @@ graph TD
 
 ## 📄 License
 
-This project is proprietary. All rights reserved.
+- Apache-2.0
++ Licensed under the Apache License, Version 2.0 (Apache-2.0). See the [LICENSE](./LICENSE) file for details.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: plus

(MD004, ul-style)